monix-execution/shared/src/main/scala_2.13-/monix/execution/compat.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2014-2019 by The Monix Project Developers.
+ * See the project homepage at: https://monix.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package monix.execution
+
+import scala.collection.generic.CanBuildFrom
+import scala.collection.mutable
+
+object compat {
+
+  type BuildFrom[-From, -A, +C] = CanBuildFrom[From, A, C]
+
+  object internal {
```

Comment:
It's better to make the whole object `package[monix]` instead of individual members.

Reply:
Done